package.json

```diff
@@ -43,18 +43,22 @@
     "node": ">=6.0.0"
   },
   "scripts": {
+    "prepublish": "rimraf lib && tsc",
```

Comment:
this generates `lib` on local install & before publish for now. Only difference is that `lib` is git ignored.